torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1626,6 +1626,11 @@ void InitXlaModuleBindings(py::module m) {
     // Register sharded tensor data.
     XLAGraphExecutor::Get()->RegisterTensor(xtensor->data());
   });
+  m.def("_xla_mark_sharding_custom_op",
```

Comment:
nit. for future reference, can we make it explicit by calling `_xla_mark_sharding_dynamo_custom_op`?

Reply:
Sg, updatd. Also added a new API in `xla_sharding.py` named `mark_sharding_dynamo_custom_op` specifically for interacting with this new custom op.